src/scripts/settings/config.js

```diff
@@ -0,0 +1,159 @@
+(function () {
```

Comment:
or perhaps, the coach specific bits can be pulled out into a separate file, leaving just the configuration logic in this file